src/Css/Style.php

```diff
@@ -1203,7 +1278,17 @@ private function evaluate_func_calc(array $rpn, float $ref_size = 0, ?float $fon
                     case '-':
                         $stack[] = $leftValue - $rightValue;
                         break;
+                    case ',':
+                        if (is_array($leftValue)) {
+                            $leftValue[] = $rightValue;
+                            $stack[] = $leftValue;
+                        } else {
+                            $stack[] = [$leftValue, $rightValue];
+                        }
+                        break;
                 }
+            } elseif (strncmp($part, '--', 2) === 0) {
```

Comment:
Okay, now I think I understand: You want to implement general CSS function parsing here. That would definitely be useful. For now, though, I would concentrate on just the math functions and ignore `var`.

We can then in the next step see how the logic here can be utilized in other places throughout the `Style` class. That would be a more ambitious task, and probably requires shuffling around a lot of code. If you want to look into that, you can open a separate PR later.

Reply:
Sorry to bother you again, can you just remove any `var`-related code for now? :-) So remove the check here, remove the test cases, and do not merge `['var']` above.

If it turns out that here is the right place to handle it, we can always add that back later as part of or as follow-up to the other PRs.